src/assets/custom/css/_sass/_website-header.scss

```diff
@@ -1,8 +1,8 @@
 .website-header {
   font-size: 13px;
-  height: 94px;
-  padding-top: 15px;
-  padding-bottom: 15px;
+  height: 74px;
```

Comment:
With the change in height there may be styles relating to internal links that need to be adjusted slightly. I know styling around internal links has been done differently in different places, 

e.g. 
https://codurance.com/services/software-modernisation/#webinar
https://codurance.com/contact-us/#get-in-touch
https://codurance.com/2017/11/16/katas-for-functional-calisthenics/#bankkata

This isn't necessarily for this branch, I just thought I'd mention it in case you're not aware

Reply:
Thanks @LL782 , we just checked and it seems only `https://codurance.com/services/software-modernisation/#webinar` is slightly misadjusted. We can make the change as part of this branch considering we're working in this area anyway.